OpenRA.Mods.Common/Widgets/SupportPowersWidget.cs

```diff
@@ -121,7 +121,7 @@ public class SupportPowerIcon
 		public void RefreshIcons()
 		{
 			icons = new Dictionary<Rectangle, SupportPowerIcon>();
-			var powers = spm.Powers.Values.Where(p => !p.Disabled);
+			var powers = spm.Powers.Values.Where(p => !p.Disabled).OrderBy(p => p.Info.SupportPowerPaletteOrder);
```

Comment:
```
			var powers = spm.Powers.Values.Where(p => !p.Disabled).OrderBy(p => p.Info.SupportPowerPaletteOrder);
```

This should prolly be added to this also https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Mods.Common/Widgets/ObserverSupportPowerIconsWidget.cs#L104